### DISCUSSION TITLE
Can't increase upload size beyond 200ish MB

### DISCUSSION THREAD

**Discussion by bluvoll** [outline/outline#6078] [Self Hosting]
Hi there, I've been using Outline for a while, but recently had the need to increase the upload size beyond 200mb, tried setting AWS_S3_UPLOAD_MAX_SIZE=1024000000 and  FILE_STORAGE_UPLOAD_MAX_SIZE=1024000000 in the env, but got this error from Formidable
FormidableError: options.maxFileSize (209715200 bytes) exceeded, received 209721736 bytes of file data
    at Stream.<anonymous> (/opt/outline/node_modules/formidable/src/Formidable.js:358:11)

I tried editing the file but didnt help either, any help would be greatly appreciated.

**Comment by tommoor** [comment]
Addressed in https://github.com/outline/outline/commit/b53c595e1bcdeab84c0dab85ca24a48e8749686b – the fix will be in the next release.
